pkgs/unified_analytics/lib/src/log_handler.dart

```diff
@@ -89,26 +111,40 @@ class LogHandler {
     final DateTime startDateTime = records.first.localTime;
     final DateTime endDateTime = records.last.localTime;
 
-    // Collection of unique sessions
+    // Map with counters for user properties
     final Map<String, Set<Object>> counter = <String, Set<Object>>{
       'sessions': <int>{},
       'flutter_channel': <String>{},
       'tool': <String>{},
     };
+
+    // Map of counters for each event
+    final Map<String, int> eventCount = <String, int>{};
     for (LogItem record in records) {
       counter['sessions']!.add(record.sessionId);
       counter['tool']!.add(record.tool);
       if (record.flutterChannel != null) {
         counter['flutter_channel']!.add(record.flutterChannel!);
       }
+
+      // Count each event, if it doesn't exist in the [eventCount]
+      // it will be added first
+      if (!eventCount.containsKey(record.eventName)) {
+        eventCount[record.eventName] = 0;
+      }
+      eventCount[record.eventName] = eventCount[record.eventName]! + 1;
     }
 
     return LogFileStats(
       startDateTime: startDateTime,
+      minsFromStartDateTime: clock.now().difference(startDateTime).inMinutes,
       endDateTime: endDateTime,
+      minsFromEndDateTime: clock.now().difference(endDateTime).inMinutes,
```

Comment:
As noted previously, I think it should be up to the client to compare these values with clock.now(), but if we leave this we should store `clock.now()` in a variable and re-use it both times, rather than calling it twice.